src/Microsoft.PowerShell.PSReadLine/ConsoleLib.cs

```diff
@@ -386,13 +403,16 @@ public static string ToGestureString(this ConsoleKeyInfo key)
                 sb.Append("Alt");
             }
 
-#if UNIX
             char c = key.KeyChar;
-#else
-            // Windows cannot use KeyChar as some chords (like Ctrl+[) show up as control characters.
-            char c = ConsoleKeyChordConverter.GetCharFromConsoleKey(key.Key,
-                (mods & ConsoleModifiers.Shift) != 0 ? ConsoleModifiers.Shift : 0);
+#if !UNIX
+            if (_toUnicodeApiAvailable)
```

Comment:
A few special cases is not over optimizing and is a practical assuming those apis aren't really generally useful.
The default output of `Get-PSReadlineKeyBindings` should not contain `Oem`.